src/AWS.Deploy.ServerMode.Client/AWS.Deploy.ServerMode.Client.csproj

```diff
@@ -11,6 +11,7 @@
     <PackageProjectUrl>https://github.com/aws/aws-dotnet-deploy</PackageProjectUrl>
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>..\..\public.snk</AssemblyOriginatorKeyFile>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
```

Comment:
Seeing that we don't want/expect users to use this package I don't think we should reuse the main README file. Either have a separate README file how we don't expect you to use this or just don't have one.

Reply:
Removed it.